docs/topics/mta-export-questionnaire.adoc

```diff
@@ -5,15 +5,15 @@
 
 :_content-type: PROCEDURE
 [id="mta-export-questionnaire_{context}"]
-= Exporting a questionnaire
+= Exporting an assessment questionnaire
 
-.Procedure
+In the {ProductName} ({ProductShortName}) 7.0, you can export an assessment questionnaire to the desired location on your system. 
```

Comment:
```suggestion
In the {ProductFullName} 7.0, you can export an assessment questionnaire to the desired location on your system. 
```
Pedantic - but we added this attribute for things like this

Reply:
Hah, yup, I was just unsure about it since I saw both options throughout the doc :-)